src/app/register/register.component.ts

```diff
@@ -0,0 +1,92 @@
+import { Component, OnInit } from '@angular/core';
+import { CouchService } from '../shared/couchdb.service';
+import { FormsModule } from '@angular/forms';
+import { FormBuilder, FormControl, FormGroup, Validators, FormControlName } from '@angular/forms';
+import { MatRadioModule , MatFormFieldModule, MatButtonModule, MatInputModule } from '@angular/material';
+import { UserData } from './UserData';
+
+
+@Component({
+  selector: 'app-register',
+  templateUrl: './register.component.html',
+  styleUrls: [ './register.component.scss' ]
+})
+export class RegisterComponent implements OnInit {
+
+  registerForm: FormGroup;
+  educationLevel = [ 1, 2, 3, 4, 5, 6 , 7, 8, 9, 11, 12, 'Higher' ];
```

Comment:
1-12 are number and one is string . Could you please change it all to the same type ?